src/ray/protobuf/gcs_service.proto

```diff
@@ -426,6 +426,18 @@ message InternalKVGetReply {
   bytes value = 2;
 }
 
+message InternalKVMultiGetRequest {
+  repeated bytes keys = 1;
+  bytes namespace = 2;
+}
+
+message InternalKVMultiGetReply {
+  GcsStatus status = 1;
+  // The key-value pairs that are found. Bytes cannot be used as a key in a map, so
+  // we use a string instead.
+  map<string, bytes> results = 2;
```

Comment:
This types here are inconsistent (Everywhere else we use `bytes` for both keys and values.) @iycheng would it be better to do this instead:
```
repeated bytes keys = 2;
repeated bytes values = 3;
```
Or does it not really matter?

Reply:
According to https://protobuf.dev/programming-guides/proto3/ string means 

> A string must always contain UTF-8 encoded or 7-bit ASCII text, and cannot be longer than 2^32.

Here we need bytes, because key can be just raw bytes.

Reply:
maybe change it to: https://protobuf.dev/programming-guides/proto3/#backwards-compatibility

where key and value are bytes?

Reply:
@iycheng Done, I used the backwards compatibility approach in your link.  Please take another look!